nemo/collections/llm/api.py

```diff
@@ -203,6 +204,9 @@ def finetune(
         optim (Optional[OptimizerModule]): The optimizer module to be used. If not provided, the default
             optimizer from the model will be used.
         peft (Optional[PEFT]): A PEFT (Parameter-Efficient Fine-Tuning) configuration to be applied.
+        tokenizer (Optional[TokenizerType]): Tokenizer setting to be applied. Can be 'data' or 'model'
+            or an instance of TokenizerSpec. If 'data' uses the data loader's tokenizer (useful if data loader
+            uses a separate tokenizer from model).
```

Comment:
Sorry, i meant something like 
"Specifying a new tokenizer (rather than the tokenizer from the model checkpoint) is useful for expanding vocabulary or adding special tokens"
If there is something else, feel free to add it too. I just thought it may be not straightforward to some users why we don't use the same tokenizer that comes from the model checkpoint.

Reply:
updated